docs/source/membership/membership.md

```diff
@@ -50,62 +94,200 @@ multiple MSPs and name them accordingly, e.g., `ORG2-MSP-NATIONAL` and
+the banking scenario above), allow the user to authenticate
+itself in its transactions as a member of a channel (e.g. in chaincode
+transactions), or as the owner of a specific role into the system (an
+org admin, for example, in configuration transactions).
 
-**Every node and user must have a local MSP defined**, as it defines who has
-administrative or participatory rights at that level (peer admins will not necessarily
-be channel admins, and vice versa).
+**Every node must have a local MSP defined**, as it defines
+who has administrative or participatory rights at that level (peer
+admins will not necessarily be channel admins, and vice versa).  This
+allows for authenticating member messages outside the context of a
+channel and to define the permissions over a particular node
+(who has the ability to install chaincode on a peer, for example).
+
+An orderer local MSP is also defined on the file system of the node
+and only applies to that node. Like peer nodes, orderers are also
+owned by a single organization and therefore have a single MSP to
+list the actors or nodes it trusts.
+
+### Channel MSPs
 
 In contrast, **channel MSPs define administrative and participatory rights at the
-channel level**. Every organization participating in a channel must have an MSP
-defined for it. Peers and orderers on a channel will all share the same view of channel
+channel level**. Peers and orderers on a channel share the same view of channel
 MSPs, and will therefore be able to correctly authenticate the channel participants.
 This means that if an organization wishes to join the channel, an MSP incorporating
 the chain of trust for the organization's members would need to be included in the
 channel configuration. Otherwise transactions originating from this organization's
-identities will be rejected.
+identities will be rejected. Whereas local MSPs are represented as a folder
+structure on the file system, channel MSPs are described in a channel
+config.json file.
+
+![MSP1d](./ChannelMSP.png)
+
+*Snippet from a channel config.json file that includes two organization
+MSPs.*
+
+**Channel MSPs identify who has permissions at a channel level**.
+
+The channel MSP defines the _relationship_ between the identities of
+channel members (which themselves are MSPs) and the enforcement of
+channel level policies. Channel MSPs contain the MSPs of the organizations of the
+channel members.
+
+**Every organization participating in a channel must have an MSP
+defined for it**. In fact, it is recommended that there is a one-to-one mapping
+between organizations and MSPs. If all network members are part of
+a single organization or MSP, data privacy is sacrificed. Multiple
+organizations facilitate privacy by segregating ledger data to only channel
+members. If more granularity is required within an
+organization, the organization can be divided into organizational units (OUs)
+as described above.
```

Comment:
I think the main point about organization MSPs is that they define who is empowered to act on behalf of the organization. This includes not just the interior configuration of the org as defined in the channel config (who the org admins are, for example), but also approving of administrative tasks in which the organization has a role (adding new members to the channel, for example). 

I know you say this below but I think it's the first/key point to highlight.